applications/GeoMechanicsApplication/tests/cpp_tests/test_convergence_criteria_factory.cpp

```diff
@@ -48,22 +50,20 @@ KRATOS_TEST_CASE_IN_SUITE(Create_ReturnsCorrectConvergenceCriteria_ForResidual,
     )";
 
     const auto convergence_criteria = ConvergenceCriteriaFactoryType::Create(Parameters{valid_parameters});
-    const auto residual_criterion = dynamic_cast<const ResidualCriteria<SparseSpaceType, LocalSpaceType>*>(convergence_criteria.get());
+    const auto residual_criterion =
+        dynamic_cast<const ResidualCriteria<SparseSpaceType, LocalSpaceType>*>(convergence_criteria.get());
     KRATOS_EXPECT_NE(residual_criterion, nullptr);
 }
 
-KRATOS_TEST_CASE_IN_SUITE(Create_Throws_WhenConvergenceCriterionDoesNotExist, KratosGeoMechanicsFastSuiteWithoutKernel)
-{
-    KRATOS_EXPECT_EXCEPTION_IS_THROWN(
-            ConvergenceCriteriaFactoryType::Create(Parameters{"{}"}),
-            "No convergence_criterion is defined, aborting.")
-}
+KRATOS_TEST_CASE_IN_SUITE(Create_Throws_WhenConvergenceCriterionDoesNotExist,
+                          KratosGeoMechanicsFastSuiteWithoutKernel){
+    KRATOS_EXPECT_EXCEPTION_IS_THROWN(ConvergenceCriteriaFactoryType::Create(Parameters{"{}"}),
+                                      "No convergence_criterion is defined, aborting.")}
 
 KRATOS_TEST_CASE_IN_SUITE(Create_Throws_WhenConvergenceCriterionIsUnknown, KratosGeoMechanicsFastSuiteWithoutKernel)
 {
     KRATOS_EXPECT_EXCEPTION_IS_THROWN(
-            ConvergenceCriteriaFactoryType::Create(Parameters{R"({"convergence_criterion" : "something_unknown" })"}),
-             "The convergence_criterion (something_unknown) is unknown, supported criteria are: 'displacement_criterion'")
+        ConvergenceCriteriaFactoryType::Create(Parameters{R"({"convergence_criterion" : "something_unknown" })"}), "The convergence_criterion (something_unknown) is unknown, supported criteria are: 'displacement_criterion'")
```

Comment:
This became a very long line, maybe it helps to put brackets around the 'call' part?

Reply:
It turned out that extracting the construction of the `Parameters` instance solved the issue of the very long line. Please, have a look yourself and see whether it is better now. Thanks you.